Examples/JS/flow/EverythingType.js

```diff
@@ -0,0 +1,61 @@
+//  @flow
+//
+//  EverythingType.js
+//  Autogenerated by plank
+//
+//  DO NOT EDIT - EDITS WILL BE OVERWRITTEN
+//  @generated
+//
+
+import type { PlankDate, PlankURI } from './runtime.flow.js';
+import type { BoardType } from './BoardType.js';
+import type { ImageType } from './ImageType.js';
+import type { PinType } from './PinType.js';
+import type { UserType } from './UserType.js';
+
+export type EverythingMapPolymorphicValuesType = UserType | BoardType | ImageType | PinType | EverythingType | Array<*> | {};
+
+export type EverythingPolymorphicPropType = UserType | BoardType | ImageType | PinType | EverythingType | string | boolean | number | number | PlankDate | PlankURI;
+
+export type EverythingIntEnum = 
+  | 1 /* int_case_1 */
+  | 2 /* int_case_2 */
+  | 3 /* int_case_3 */
+;
+
+export type EverythingStringEnum = 
+  | 'case1'
+  | 'case2'
+  | 'case3'
+;
+
+export type EverythingType = $Shape<{|
+  +array_prop: ?Array<*>,
+  +boolean_prop: boolean,
+  +date_prop: ?PlankDate,
+  +int_enum: EverythingIntEnum,
+  +int_prop: number,
+  +list_polymorphic_values: ?Array<*>,
+  +list_with_object_values: ?Array<string>,
+  +list_with_other_model_values: ?Array<UserType>,
+  +list_with_primitive_values: ?Array<number /* Integer */>,
+  +map_polymorphic_values: ?{ +[string]: EverythingMapPolymorphicValuesType },
+  +map_prop: ?{},
+  +map_with_object_values: ?{ +[string]: string },
+  +map_with_other_model_values: ?{ +[string]: UserType },
+  +map_with_primitive_values: ?{ +[string]: number } /* Integer */,
+  +number_prop: number,
+  +other_model_prop: ?UserType,
+  +polymorphic_prop: ?EverythingPolymorphicPropType,
+  +set_prop: ?Set<*>,
```

Comment:
We have to look into if this will actually work. I think if we pass that through we will just have an `Array<*>`.

Reply:
After looking into it, we should change this to an `Array<*>`. Every `NSSet` should be an `Array` in json / js and every `NSDictionary` an Object / {} in json / js.

Reply:
Done - Let me know if you get to re-test this with the latest patch

Reply:
Can you regenerate the Examples files please.